marketplace/bynder-dialog/src/index.js

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useReducer } from 'react';
+import PropTypes from 'prop-types';
+import ReactDOM from 'react-dom';
+import { Button } from '@contentful/forma-36-react-components';
+import { init, locations } from 'contentful-ui-extensions-sdk';
+import '@contentful/forma-36-react-components/dist/styles.css';
+import './index.css';
+import extension from '../extension.json';
+import BynderDialog from './BynderDialog';
+import Thumbnail from './Thumbnail';
+
+function reducer(state, action) {
+  switch (action.type) {
+    case 'add-all-assets':
+      return [...state, ...action.payload];
+    case 'remove-asset':
+      return state.filter(item => item.id !== action.payload);
+    default:
+      return state;
+  }
+}
+
+function Field({ sdk }) {
+  const [assets, dispatch] = useReducer(reducer, sdk.field.getValue() || []);
+
+  useEffect(() => {
+    sdk.window.startAutoResizer();
```

Comment:
I noticed this already in `create-contentful-extension`. Is there any advantage to doing it in the React lifecycle instead of doing it in init (before rendering)?

Reply:
In this particular case I don't want it to be called in Dialog extension and I use `updateHeight` there.